src/test/java/uk/gov/hmcts/reform/pip/data/management/controllers/publication/PublicationControllerTest.java

```diff
@@ -518,4 +521,28 @@ void testArchiveArtefact() {
                      response.getBody(), "Response from archiving does not match expected message"
         );
     }
+
+    @Test
+    void testInvalidLcsuArtefactTypeForProdEnvironmentSendsBadRequest() {
+        when(validationService.validateHeaders(any())).thenReturn(lcsuHeaders);
+
+        // Ensure the environment field is set to "prod"
+        ReflectionTestUtils.setField(publicationController, "environment", "prod");
+
+        LcsuArtefactNotSupportedException exception = null;
+
+        try {
```

Comment:
This can be done through assertThrows which removes the need for the try/catch and extra assertion below

Reply:
done.